apps/backend/src/modules/quiz/quiz.controller.ts

```diff
@@ -92,9 +95,9 @@ class QuizController extends BaseController {
 	 *             type: object
 	 *             properties:
 	 *               answerIds:
-	 *                 type: number[]
-	 *
-	 *
+	 *                 type: array
+	 *                 items:
```

Comment:
This shouldn't be a part of your PR, avoid unnecessary changes

Reply:
It's part of issue #286 bug fix. I didn't change the implementation, only the docs.